### PR TITLE
Server Side Processing

### PR DESCRIPTION
**Pull request by himanshu2047**
Fixes convict-git#53

### CODE REVIEW

**Comment by himanshu2047** [comment]
@tymefighter ready to be reviewed

**Comment by tymefighter** [comment]
I'll try to do by today

File: src/processing/handlers/ObjectHandler.java [hunk 1]
```diff
@@ -15,8 +16,9 @@
 
 public class ObjectHandler implements INotificationHandler{
 	
-	public void onMessageReceived(String message) {
+	public static void handleBoardObject(String message) {
 		
+
 		try {
 				BoardObject boardObject = (BoardObject)Serialize.deSerialize(message);
 				boardObject.getOperation();
```
Comment (tymefighter) [inline src/processing/handlers/ObjectHandler.java:24]: What are you doing here...?

Comment (tymefighter) [reply src/processing/handlers/ObjectHandler.java:24]: You must also call the actual operation after retrieving what operation is to be performed

Comment (tymefighter) [reply src/processing/handlers/ObjectHandler.java:24]: You must also perform the operation that you retrieved

Comment (tymefighter) [reply src/processing/handlers/ObjectHandler.java:24]: Also don't forget to check if the object is sent from the same user (by looking at user ID), else you would perform it twice....

File: src/processing/handlers/BoardStateHandler.java [hunk 2]
```diff
@@ -15,15 +16,23 @@
...
-		
+	
+	public static void handleBoardState(String message) {
 		try {
-				BoardState boardState = (BoardState)Serialize.deSerialize(message);
-				ClientBoardState.maps = boardState;
+			BoardState boardState = (BoardState)Serialize.deSerialize(message);
+			ClientBoardState.maps = boardState;
 		} catch (ClassNotFoundException e) {
 			//Log the exception
```
Comment (tymefighter) [inline src/processing/handlers/BoardStateHandler.java:25]: You wrote but did not log it !!

File: src/processing/server/main/Requests.java [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+				LogLevel.SUCCESS, 
+				"Successfully sent the request for existing board from client to the server"
+		);
+		
+	}
+	
+	/**
+	 * This function will be called by a board server when every client gets disconnected from the 
+	 * server to stop the board server.
+	 * @param boardId baordID of the board server which needs to be stopped
```
Comment (tymefighter) [inline src/processing/server/main/Requests.java]: board not baord

File: src/processing/ClientBoardState.java [hunk 4]
```diff
@@ -14,61 +17,232 @@
...
+	 * communications from the server
+	 */
 	public static Port portNumber;
 	
-	//public static Map <Identifier identifier, IChanges handler>
-	
+	/** boardId will store the Board ID of the board to which
+	 * the client is connected to.
+	 */
 	public static BoardId boardId;
```
Comment (tymefighter) [inline src/processing/ClientBoardState.java:35]: I think you have to make this synchronized

File: src/processing/handlers/ObjectHandler.java (hunk 1 above)
Comment (himanshu2047) [reply src/processing/handlers/ObjectHandler.java:24]: This is satchit's file, he will complete it, it's not completed yet.

File: src/processing/ClientBoardState.java (hunk 4 above)
Comment (himanshu2047) [reply src/processing/ClientBoardState.java:35]: Why only once it will be set

File: src/processing/handlers/BoardStateHandler.java (hunk 2 above)
Comment (himanshu2047) [reply src/processing/handlers/BoardStateHandler.java:25]: This is satchit's file, see the author, I just made some changes

**Review by tymefighter — APPROVED** [review]
Gone through it, seems fine
Please Complete Satchit's Codes later

**Pull request merged**